include/date/tz.h

```diff
@@ -786,7 +786,7 @@ class time_zone
     std::vector<detail::zonelet>         zonelets_;
 #endif  // !USE_OS_TZDB
     std::unique_ptr<std::once_flag>      adjusted_;
-
+    std::string                          posix_timezone_;
```

Comment:
No getting around storing extra data here. HH gets around this by making assumptions about the TZIF format used by OS's that would increase the size of our embedded data.